<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckTools</class>
 <widget class="QWidget" name="CheckTools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>392</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>495</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>495</width>
    <height>392</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>校验工具</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/start.ico</normaloff>:/images/start.ico</iconset>
  </property>
  <widget class="QPushButton" name="CuSum8Btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>8位累加和</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Xor8Btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>8位异或</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Crc8Btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CRC8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Crc16Btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>295</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CRC16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Crc32Btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>CRC32</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CuSum16Btn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>75</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>16位累加和</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="InputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>371</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="OutputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>371</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="inputlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输入(仅支持HEX方式)：</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>输出：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="CustomCheckBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>自定义CRC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FormulaLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="InitValueLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="crclabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>简式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="initValuelabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>初值：</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
